### PR TITLE
remove kvstore::ResultCode

### PR DESCRIPTION
**Pull request by panda-sheep**
Only For removing `kvstore::ResultCode`
https://github.com/vesoft-inc/nebula-graph/pull/1022 and https://github.com/vesoft-inc/nebula-storage/pull/448 reply on this pr.

### CODE REVIEW

**Review by monadbobo — DISMISSED** [review]
LGTM.

**Comment by panda-sheep** [comment]
Replace this pr with https://github.com/vesoft-inc/nebula-common/pull/517，so close

File: src/common/interface/common.thrift [hunk 1]
```diff
@@ -200,3 +200,121 @@ struct LogInfo {
...
 }
+
+
+/*
+ * ErrorCode for graphd, metad, storaged,raftd
+ * -1xxx for graphd
+ * -2xxx for metad
+ * -3xxx for storaged
+ */
+enum ErrorCode {
```
Comment (Shylock-Hg) [inline src/common/interface/common.thrift:211]: We'd better value the number in hex format that we could easy check the type of error code by bit operatiion.

Comment (panda-sheep) [reply src/common/interface/common.thrift:211]: Why need bit operations to check the type of error code?

**Comment by panda-sheep** [comment]
Ready to review again.

File: src/common/interface/common.thrift (hunk 1 above)
Comment (Shylock-Hg) [reply src/common/interface/common.thrift:211]: To check where the error from

Comment (panda-sheep) [reply src/common/interface/common.thrift:211]: There will be a special pr to unify the error code later.
It can be ignored here for now.

**Review by critical27 — APPROVED** [review]
Good job!

**Comment by panda-sheep** [comment]
fix rebase master build failed

**Review by bright-starry-sky — APPROVED** [review]
LGTM

**Review by critical27 — APPROVED** [review]

**Pull request merged**